### ISSUE TITLE
[REQUEST] Document Required Token Access Scope per Feature

### ISSUE THREAD

**Issue by LesterCovax** [sindresorhus/refined-github#2693]
Currently, the link to create a PAT for Refined Twitter encompasses the entire `Repo` scope.  There is [currently a link provided](https://github.com/sindresorhus/refined-github/search?q=libs/api) within the extension's options to the functions accessing the API, but it's not immediately clear which access scope each one requires (i.e. `repo:status`, `repo_deployment`, `public_repo`, `repo:invite`).

I think it would be ideal to document these specifics (and even the need for a Personal Access Token) in the [README.md](https://github.com/sindresorhus/refined-github/blob/master/readme.md), instead of users finding out once they open the extension options for the first time.  Adding the required scope after each feature requiring it would be nice for those that like to limit PAT auth access, without them having to dig through the source code.  I realize that the `Repo` scope isn't very invasive, but it would have been nice to know and have documented up front.

Cheers!

**Comment by yakov116** [comment]
Duplicate of #2679

**Comment by LesterCovax** [comment]
@yakov116 How can an **issue** be a duplicate of a **_comment_** within an unrelated issue? That's not how that works.  
(I did search for similar issues before posting FWIW)

**Comment by fregante** [comment]
@LesterCovax It's duplicate in the sense that both are about documentation around PAT.

I've been hesitant to add more content on the options page, but since I had to explain the PAT in the issues before, perhaps we can add a short text near the field, either via expandable `details` or linked to a more explanatory document on the repo or issues.

When installing the extension, https://github.com/sindresorhus/refined-github/issues/1137 opens. Recently I added a small paragraph suggesting to add the token.

>  There is [currently a link provided](https://github.com/sindresorhus/refined-github/search?q=libs/api) within the extension's options to the functions accessing the API, but it's not immediately clear which access scope each one requires.

None of those extra permissions are required, but the all-inclusive `repo` permission is the only way to gain access to private repos, so we have to select it.
